### PR TITLE
[WIP] Use 4.10 stable to 4.15 stable upgrade for multi hop test

### PR DESCRIPTION
**Pull request by JianLi-RH**

### CODE REVIEW

**Comment by JianLi-RH** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-stable-4.15-upgrade-from-stable-4.10-gcp-ipi-f14

**Comment by JianLi-RH** [comment]
/cc @shellyyang1989 @jiajliu @jianlinliu 

```
[36mINFO[0m[2023-11-14T13:30:31Z] Running step gcp-ipi-f14-cucushift-chainupgrade-toimage. 
[36mINFO[0m[2023-11-14T18:28:54Z] Step gcp-ipi-f14-cucushift-chainupgrade-toimage succeeded after 4h58m22s.

1 failures in upgrade prepare
1 failures in upgrade check
```

**Comment by jianlinliu** [comment]
why would use 4.15 stable build to replace nightly build?

**Comment by shellyyang1989** [comment]
> why would use 4.15 stable build to replace nightly build?

By reviewing our chain upgrade jobs, I'm seeing all are on nightly. We are lack of coverage on stable chain upgrade. So I proposed to move some nightly chain upgrade to stable.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.10.yaml [hunk 1]
```diff
@@ -54,19 +54,20 @@ releases:
...
   target:
-    candidate:
+    prerelease:
       architecture: amd64
       product: ocp
-      stream: nightly
-      version: "4.15"
+      version_bounds:
+        lower: 4.14.0-0
+        upper: 4.15.0-0
```
Comment (shellyyang1989) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.10.yaml:62]: We need a 4.15 ec as the target. Suggest to chage:

```
target:
    release:
      architecture: amd64
      channel: candidate
      version: "4.15"
```

**Comment by jianlinliu** [comment]
> chain upgrade jobs, I'm seeing all are on nightly. We are lack of coverage on stable chain upgrade. So I proposed to move some nightly ch

If that, we should create a dedicated stable (or ec) chain upgrade ci yaml, but not rename the existing nightly one, right?

**Comment by JianLi-RH** [comment]
> > chain upgrade jobs, I'm seeing all are on nightly. We are lack of coverage on stable chain upgrade. So I proposed to move some nightly ch
> 
> If that, we should create a dedicated stable (or ec) chain upgrade ci yaml, but not rename the existing nightly one, right?

Creating new config sounds good to me, then we can also add new profile.

**Pull request closed without merging**